test/chain.jl

```diff
@@ -0,0 +1,85 @@
+
+import Flux, Fluxperimental
+
```

Comment:
```suggestion
```
This file could use a general whitespace and dead comment removal pass, I think.

Reply:
Just added new commit which goes through and cleans up the test file.